### ISSUE TITLE
Allow customization of member IDs

### ISSUE THREAD

**Issue by nguyer** [kaleido-io/firefly-cli#14]
We use a string for member identifiers throughout the codebase. However, right now the strings are generated by simply incrementing a counter as each member's config is generated. Users should be able to customize the name of each member both interactively and by passing flags.

**Comment by tobigiwa** [comment]
Just to clarify, which of the identifers would you like to be customizable.

`"members": [
  {
   "id": "0",
   "index": 0,
   "account": {
    "name": "org_32ec44",
    "orgName": "org_32ec44"
   },
   ....
   "orgName": "org_32ec44",
   "nodeName": "node_32ec44",
   "namespaces": null
  },]`
  
  is it the "name" and "orgName" in the account...OR the "orgName" outside it? am betting it just the "name" in accounts.

**Comment by tobigiwa** [comment]
just to clarify, you mean "OrgName"?

**Comment by nguyer** [comment]
No, OrgName is separate from ID: https://github.com/hyperledger/firefly-cli/blob/main/pkg/types/organization.go#L20

Though in reality, now that the org name can be customized, I'm not sure how much value this feature has anymore

**Comment by tobigiwa** [comment]
Hmmm... should I let it sleep and move on to another instead?
